### PR TITLE
Version check

### PR DESCRIPTION
**Pull request by unnonouno**
This patch removes unsupported version combinations based on installed chainer/cupy version.
merge #290 first.

### CODE REVIEW

**Comment by okuta** [comment]
Please rebase this

File: version.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import os
+import re
+
+
+def get_version(setup_path):
+    with open(setup_path) as f:
+        for line in f:
+            m = re.match(
+                ' *version=\'([0-9]+)\\.([0-9]+)\\.([0-9]+)([a-c][0-9]?)?\'',
```
Comment (okuta) [inline version.py]: Please support rc version.

**Comment by unnonouno** [comment]
I updated

**Comment by okuta** [comment]
LGTM!

**Pull request merged**